required/latex-lab/build.lua

```diff
@@ -17,11 +17,54 @@ installfiles = {
 		 "glyphtounicode-cmex.tex",
          "tagpdf-ns-latex-lab.def"  
 		}
-typesetfiles = {
-                 "latex-lab-*.dtx",
-                 "*-doc.tex",
-		 "*-code.tex",
-	       }
+typesetfiles_list = {
+  {
+    "blocks-code.tex",
+    "blocks-doc.tex",
+    "documentmetadata-support-code.tex",
+    "documentmetadata-support-doc.tex",
+    "latex-lab-block.dtx",
```

Comment:
I know this was generated before but isn't that in fact the same as block-code.pdf?